spec.bs

```diff
@@ -1255,14 +1253,13 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
   1. Let |perBuyerExperimentGroupIds| be |auctionConfig|'s
     [=auction config/per buyer experiment group ids=].
   1. If |perBuyerExperimentGroupIds| is not null and |perBuyerExperimentGroupIds|[|buyer|]
-    [=map/exists=]:
-    1. Set |buyerExperimentGroupId| to |perBuyerExperimentGroupIds|[|buyer|].
+    [=map/exists=], then set |buyerExperimentGroupId| to |perBuyerExperimentGroupIds|[|buyer|].
   1. <dfn>Apply interest groups limits to prioritized list</dfn>:
     1. Let |buyerGroupLimit| be |allBuyersGroupLimit|.
     1. Let |perBuyerGroupLimits| be |auctionConfig|'s
       [=auction config/per buyer group limits=].
-    1. If |perBuyerGroupLimits| is not null and |perBuyerGroupLimits|[|buyer|] exists:
-      1. Set |buyerGroupLimit| to |perBuyerGroupLimits|[|buyer|].
+    1. If |perBuyerGroupLimits| is not null and |perBuyerGroupLimits|[|buyer|] exists, then set

```

Comment:
nit: maybe link to `[=map/exists=]` here? Or can happen in follow-up.

Reply:
good catch. Done.